openassessment/xblock/staff_area_mixin.py

```diff
@@ -394,17 +395,18 @@ def get_student_info_path_and_context(self, student_username, expanded_view=None
         workflow_cancellation = self.get_workflow_cancellation_info(submission_uuid)
 
         context.update({
+            'expanded_view': expanded_view,
+            'example_based_assessment': example_based_assessment,
+            'self_assessment': self_assessment,
+            'peer_assessments': peer_assessments,
+            'submitted_assessments': submitted_assessments,
+            'staff_assessment': staff_assessment,
             'score': workflow.get('score'),
             'workflow_status': workflow.get('status'),
             'workflow_cancellation': workflow_cancellation,
-            'peer_assessments': peer_assessments,
-            'submitted_assessments': submitted_assessments,
-            'self_assessment': self_assessment,
-            'example_based_assessment': example_based_assessment,
-            'expanded_view': expanded_view,
```

Comment:
These weren't in any order that I could tell, so I re-arranged them to be in roughly the order they're defined above.